libs/testserver/pipelines.go

```diff
@@ -94,3 +94,104 @@ func (s *FakeWorkspace) PipelineUpdate(req Request, pipelineId string) Response
 		Body: pipelines.EditPipelineResponse{},
 	}
 }
+
+func (s *FakeWorkspace) PipelineStartUpdate(req Request, pipelineId string) Response {
+	defer s.LockUnlock()()
+
+	_, exists := s.Pipelines[pipelineId]
+	if !exists {
+		return Response{
+			StatusCode: 404,
+			Body:       map[string]string{"message": fmt.Sprintf("The specified pipeline %s was not found.", pipelineId)},
+		}
+	}
+
+	// Parse the request body to extract flags
+	var startUpdate pipelines.StartUpdate
+	if len(req.Body) > 0 {
+		err := json.Unmarshal(req.Body, &startUpdate)
```

Comment:
Why do you need this? It does not seem like you read the values of startUpdate later